python/sources/InfluxDB/main.py

```diff
@@ -32,95 +55,78 @@ def interval_to_seconds(interval):
     else:
         raise ValueError(f"Unknown interval unit: {unit}")
 
-
-# should the main loop run?
-# Global variable to control the main loop's execution
-run = True
-
-# Quix provides automatic credential injection for the client.
-# However, if needed, the SDK token can be provided manually.
-# Alternatively, you can always pass an SDK token manually as an argument.
-client = qx.QuixStreamingClient()
-
-# Initializing the Quix output topic for data streaming
-print("Opening output topic")
-producer_topic = client.get_topic_producer(os.environ["output"])
-
-# Creating a new data stream for Quix
-# A stream represents a collection of data belonging to a single session of a source.
-# A stream is a collection of data that belong to a single session of a single source.
-stream_producer = producer_topic.create_stream()
-# Editing the properties of the stream
-# Assigning a stream ID allows for appending data to the stream later on.
-# stream = producer_topic.create_stream("my-own-stream-id")  # To append data into the stream later, assign a stream id.
-stream_producer.properties.name = "influxdb-query"  # Give the stream a human readable name (for the data catalogue).
-stream_producer.properties.location = "/influxdb"  # Save stream in specific folder to organize your workspace.
-stream_producer.properties.metadata["version"] = "Version 1"  # Add stream metadata to add context to time series data.
-
-
-
-client = InfluxDBClient3.InfluxDBClient3(token=os.environ["INFLUXDB_TOKEN"],
-                         host=os.environ["INFLUXDB_HOST"],
-                         org=os.environ["INFLUXDB_ORG"],
-                         database=os.environ["INFLUXDB_DATABASE"])
-
-
-measurement_name = os.environ.get("INFLUXDB_MEASUREMENT_NAME", os.environ["output"])
-interval = os.environ.get("task_interval", "5m")
 interval_seconds = interval_to_seconds(interval)
 
 # Function to fetch data from InfluxDB and send it to Quix
 # It runs in a continuous loop, periodically fetching data based on the interval.
 def get_data():
-
     # Run in a loop until the main thread is terminated
     while run:
         try:
-            # Query InfluxDB 3.0 usinfg influxql or sql
-            table = client.query(query=f'SELECT * FROM "{measurement_name}" WHERE time >= now() - {interval}', language="influxql")
+            # Query InfluxDB 3.0 using influxql or sql
+            table = client.query(query=f'SELECT * FROM "{measurement_name}" WHERE time >= now() - {interval}',
+                                 language="influxql")
+
+            # Convert the result to a pandas dataframe. Required to be processed through Quix.
+            influx_df = table.to_pandas().drop(columns=["iox::measurement"])
```

Comment:
Relates to some of my other previous comments: do we need DF's here in this case (in this case I am unsure since I am not familiar with how InfluxDB works)?